lib/node_modules/@stdlib/blas/ext/base/dnansumors/lib/ndarray.native.js

```diff
@@ -32,8 +30,8 @@ var addon = require( './dnansumors.native.js' );
 *
 * @param {PositiveInteger} N - number of indexed elements
 * @param {Float64Array} x - input array
-* @param {integer} stride - stride length
-* @param {NonNegativeInteger} offset - starting index
+* @param {integer} strideX - index increment
```

Comment:
I'd say either stick with `stride length` or use `index increment` everywhere within this package.

Reply:
If we have currently have different conventions for ndarray files and other files, that discrepancy should be fixed (e.g., in a separate PR).

Reply:
We should then stick with `index increment` since that directly describes how much the index increments as you move to the next element. I think `stride length` is more abstract in its meaning.

Reply:
I suggest doing some quick codebase analytics to see which is the more common convention. Having a quick look through `blas/base`, it seems we typically use, e.g., "`x` stride length", but this may be sampling bias.

We use the "stride" terminology fairly heavily, so I am okay with `stride length`. The main thing is that we should be more consistent.

Reply:
Yeah, `stride length` is used `blas/base`. 

Going through the `blas/ext/base`, I see `index increment` is used for both ndarray and "normal" files. 

I don't remember where this `stride length` for "normal" files and `index increment` ndarray files came from. It may have been a mistake in one package that got carried over in subsequent PRs.

I'll make it `stride length` in this and other open PRs.